docker-compose.yml

```diff
@@ -5,7 +5,7 @@
 version: '2'
 services:
   build-requirements:
-    image: python:3.9
+    image: python:3.9  # NOTE: this should be updated to match what is in ./Dockerfile
```

Comment:
I meant for you to add the comment to `Dockerfile`, since that is more likely to be updated without appropriately updating this file.

Reply:
Can't we just add the lines from below to this job?

```yaml
    build:
      context: ./
      dockerfile: ./Dockerfile-dev
```

Reply:
Yeah but that builds the entire image which we don't need -- but we could still accomplish this without building the entire image by having a base target which is just the Python image.

Reply:
Ah, I misunderstood your comment - what you're suggesting is exactly what I was hoping for. :)